lamin_cli/_settings.py

```diff
@@ -11,9 +11,9 @@
 @click.group(invoke_without_command=True)
 @click.pass_context
 def settings(ctx):
-    """Manage settings.
+    """Display or modify LaminDB configuration settings.
 
-    Call without subcommands and options to show settings.
+    Run without additional arguments to show current values.
```

Comment:
I think people `call functions` but `run CLI commands`?

Reply:
I'd say "call" because of the confusion with a "run". Otherwise good.